src/contributors/SwimLaneContributor.ts

```diff
@@ -80,7 +80,7 @@ export class SwimLaneContributor extends Contributor {
     /**
      * The range of data that this contributor fetches.
      */
-    public range: RangeResponse;
+    public range: ComputationResponse;
```

Comment:
Shouldn't we also remove any reference to `range` variable name and comment)?

Reply:
range variable will be used for DetailedHistogram in the toolkit (a PR for that is coming)